### PR TITLE
Convert districts.md.erb to districts.md.partial

### PR DESCRIPTION
**Pull request by nkiruka**

### CODE REVIEW

File: pegasus/sites.v3/hourofcode.com/public/how-to/districts.md.partial [hunk 1]
```diff
@@ -13,26 +13,26 @@ nav: how_to_nav
 ## Get every school participating, every student learning!
 
 ### 1. Recruit local schools
-Share [this email](<%= resolve_url('/promote/resources#sample-emails') %>) and [teacher how to guide](<%= resolve_url('/how-to') %>), or include a [short blurb](<%= resolve_url('/promote/stats') %>) in newsletters/district communications.
+Share [this email]({{ promote/sample_emails}}) and [teacher how to guide]({{ urls/how_to_guide }}), or include a [short blurb]({{ promote/stats_url }}) in newsletters/district communications.
```
Comment (bethanyaconnor) [inline pegasus/sites.v3/hourofcode.com/public/how-to/districts.md.partial]: nit: add a space between `promote/sample_emails` and the end curly brackets.

File: pegasus/sites.v3/hourofcode.com/views/promote/resources_social [hunk 2]
```diff
@@ -0,0 +1 @@
+<%= resolve_url('/promote/resources#social') %>
```
Comment (bethanyaconnor) [inline pegasus/sites.v3/hourofcode.com/views/promote/resources_social]: nit: this file should be name `resources_social.erb`.

**Review by bethanyaconnor — COMMENTED** [review]
A couple of nits but looks great!

File: pegasus/sites.v3/hourofcode.com/public/how-to/districts.md.partial (hunk 1 above)
Comment (nkiruka) [reply pegasus/sites.v3/hourofcode.com/public/how-to/districts.md.partial]: Fixed, thanks

**Review by bethanyaconnor — APPROVED** [review]

**Pull request merged**